### PR TITLE
Add Milestone-1 Header/Footer with bootstrap

### PR DESCRIPTION
**Pull request by garciajordy**
Add everything of Milestone-1 from week-3
- Add header and style with bootstrap
- Add footer and style  with bootstrap
- Edit README.md for milestone-1

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello, @garciajordy  and @KabohaJeanMark. Good job guys, 👏 👏 👏

## The highlights: 
👍 The project is almost looking identical.
👍 The Bootstrap installation is correct.
👍 The git-flow is correct.
👍 The `linters` is correctly set-up.
👍  Good work in the live link.
👍 Nice PR message, title and commit message.

## What to do next:  
**Please check my required points listed below:** 

**[REQUIRED]**
 - [x]  In the `Header` the `LOGIN` and `SUBSCRIBE` are links. 
 - [x]  In the `Footer` the social media icons change size with the changing screen sizes.
 - [ ]  In the CSS file, I can see you have declared variables for colors and then not using it correctly, please use the variables correctly.
 - [ ]  You are using `border-bottom: rgb(190, 190, 190) solid 1px;` twice, please use variables instead.
 - [x]  Use a dedicated folder for all the images, SVG files and name its assets, and place them in the root directory.
 - [ ]  I wonder why you used the package.json file, you don't need it.
 - [x]  In your `Readme` file, add instructions for how a user can download and run the project locally.
 ##
**Please make the changes and ask for a re-review.**

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

**Comment by garciajordy** [comment]
Hi :wave: @LamiaSristy ,

Thank you for your feedback! 
We have learned from the feedback and appreciate the time you took to give us this amazing feedback.

We have worked on almost all the feedback the only thing left out is that we actually are not required to use SASS so we only focused on using bootstrap as that was the main task for this project.

Kind Regards

File: style.css [hunk 1]
```diff
@@ -40,3 +40,127 @@ a:hover {
...
+  border-bottom: rgb(190, 190, 190) solid 1px;
+}
+
+.middle-nav a {
+  color: black;
+  border-right: rgb(190, 190, 190) solid 1px;
+}
+
+.last-link {
+  border: none !important;
```
Comment (umairarshadbutt) [inline style.css]: Remove important

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @garciajordy and @KabohaJeanMark 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [ ] Remove unnecessary files like `package.json` for this project
- [x] `Newsweek` logo should have a link
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by garciajordy** [comment]
Thanks for the feedback, we have solved all the issues by adding links to the logo and removing important from CSS, and deleting the package.json.

However, when we delete the package.json we get linter errors...

File: style.css (hunk 1 above)
Comment (KabohaJeanMark) [reply style.css]: Thanks. We have removed this and used an alternative with id tag styling.

**Comment by mimipeshy** [comment]
### STATUS :APPROVED :partying_face: :champagne: 

Hey team @garciajordy  and @KabohaJeanMark, you guys have done an amazing job with the feedback given. Your page looks similar to the original page. This milestone is therefore approved. Go ahead and merge

All the best in the next milestones. Happy Coding :computer: 

Regards
Peris Ndanu

**Pull request merged**